### PR TITLE
Fix 159 s3 documentation

### PR DESCRIPTION
**Pull request by rizart**
Full described functionality will be implemented upon closing https://github.com/rucio/rucio/issues/5451.

### CODE REVIEW

File: docs/installing_server.md [hunk 1]
```diff
@@ -626,3 +626,89 @@ that were described previously. The following steps are needed:
         ISS=https://auth.cern.ch/auth/realms/cern" \
     --email "rucio@cern.ch"
     ```
+
+## S3 Storage Configuration (FTS3 & GFAL2)
```
Comment (bari12) [inline docs/installing_server.md:630]: I think this should be it's own article in the documentation. (Under Operator documentation)

Comment (rizart) [reply docs/installing_server.md:630]: Indeed, fits better there, moving it!

**Comment by rizart** [comment]
`s3_rse_config.md` as a filename breaks the `check-file-names.sh` check because of the `^[a-z_]*\.md$` regexp. Should I change it to include numbers or should I change the filename?

**Comment by bari12** [comment]
Hi @rizart 
Sorry I missed your last message. Please change the regexp and rebase the PR, otherwise this is good to go.

**Pull request merged**